### ISSUE TITLE
[need_help]

### ISSUE THREAD

**Issue by warioishere** [lightningnetwork/lnd#8477]
### Background

Describe your issue here.

I am trying to restore a single channel from a single backup file. It does not work, see details below.

### Your environment

* version of `lnd`
lnd 0.17.4 rc1 (containerized by btcpay-docker)
* which operating system (`uname -a` on *Nix)
Linux btcpay 6.5.11-7-pve #1 SMP PREEMPT_DYNAMIC PMX 6.5.11-7 (2023-12-05T09:44Z) x86_64 x86_64 x86_64 GNU/Linux

* version of `btcd`, `bitcoind`, or other backend
26.0
* any other relevant environment details
Running the node on Proxmox

### Steps to reproduce

Trying to export a specific channel with `exportchanbackup --chan_point 9110b04f1ec66f20e3d7a8e0a8e15ce10f0a4c8729872c3a5cd036093829ef4b:1 --output_file=/root/.lnd/bitcoinsuisse.backup
`
and restoring with

```
**restorechanbackup --single_file=/root/.lnd/bitcoinsuisse.backup**
[lncli] unable to restore chan backups: rpc error: code = Unknown desc = unable to unpack single backups: could not derive private key for legacy channel revocation root format: unable to derive private key
```

### Expected behaviour

channel will will be restored from single backup file

### Actual behaviour

Does not work:

[lncli] unable to restore chan backups: rpc error: code = Unknown desc = unable to unpack single backups: could not derive private key for legacy channel revocation root format: unable to derive private key

Additional Info: the restore was already triggered with a full restorechanbackup some months ago, but it did not trigger a forceclose on this specific channel on the other side propably due to high fee enviroment that time.

**Comment by guggero** [comment]
Are you exporting the backup from the node you initially ran `restorechanbackup` with the full file on? Do you still have the full file available?

**Comment by warioishere** [comment]
> Are you exporting the backup from the node you initially ran `restorechanbackup` with the full file on? Do you still have the full file available?

Hello Oli

yes its the same node,maybe you can remember the issue I had with my SCB case we have discussed arround Christmas/Newyear

I never changed or touched the original SCB file so I guess its the original SCB file, I continued using the same node normally since then.

Greetings

**Comment by guggero** [comment]
Yes, I remember. And I think the issue is indeed that you're exporting a SCB from an SCB which is IMO not fully supported.
What you can do instead is take the original file and use [`chantools filterbackup`](https://github.com/lightninglabs/chantools/blob/master/doc/chantools_filterbackup.md) to reduce the file to one that only has the channel in question (meaning filter out all other channels). Then use that (as a `multi_file` but with just one entry).

**Comment by guggero** [comment]
But I'm not sure doing so would change anything. If you say the node you used the full SCB on is still running like that, then it should periodically try and request the peer to force close. Or at least that's what I think it should. Maybe it only happens on startup though. So probably still worth a try, just want to manage expectations.

**Comment by warioishere** [comment]
> Yes, I remember. And I think the issue is indeed that you're exporting a SCB from an SCB which is IMO not fully supported. What you can do instead is take the original file and use [`chantools filterbackup`](https://github.com/lightninglabs/chantools/blob/master/doc/chantools_filterbackup.md) to reduce the file to one that only has the channel in question (meaning filter out all other channels). Then use that (as a `multi_file` but with just one entry).

I understand the issue when you told me that I exported a channelback from an already restored scb channel. 
I will try chantools later. 

Do you think chantools triggerforce might do the trick?

I have contact with bitcoinsuisse, they cannot manually force close, they say they have the issue that says `channel already closed `  when they try to force close again. 
I dont get much information because I talk to a relation manager which passed me few infos from the IT department. I told them already to raise mempool, restart their node afterwards and do this and that. But I get an answer with almost no Infos once in a week.
